file_folder/test1/c.txt

```diff
@@ -0,0 +1 @@
+a s e f d df d s s d s s df d c cv f f f f ds sss rr
```

Comment:
You can use `Lorem Ipsum` [generator](https://www.lipsum.com/) for creating sample files.